main.tf

```diff
@@ -517,6 +517,21 @@ resource "azapi_resource" "container_app_job_agentless" {
   })
 }
 
+# Trigger execution, if requested
+resource "null_resource" "job_execution_now" {
```

Comment:
nit: it is apparently recommended to use `terraform_data` instead of `null_resource` 
e.g https://devdosvid.blog/2023/04/16/hello-terraform-data-goodbye-null-resource/